src/java/org/apache/cassandra/db/ColumnFamilyStoreMBean.java

```diff
@@ -217,6 +217,12 @@ public List<String> importNewSSTables(Set<String> srcPaths,
      */
     public long[] getPerLevelSizeBytes();
 
+    /**
+     * @return sstable count for each bucket in TWCS. null unless time window compaction is used.
+     *         array index corresponds to bucket(int[0] is for most recent, ...).
```

Comment:
Given its temporality, I might be inclined to want to read oldest to newest left to right. Is the thought here to emulate what we do w/ LCS where the levels move left to right? (i.e. The age of the data moves left to right?)

Reply:
I can't speak to the intent of the original author (will update commit message to reflect that it was Stefan's work; can check w/him), however the current shape of things is recency left to right, levels left to right, level 0 corresponding to "newest".

```
array index corresponds to level(int[0] is for level 0, ...)

array index corresponds to bucket(int[0] is for most recent, ...).
```

I don't really have an opinion on it excepting the current formatting provides logical symmetry between the two JMX calls and I like consistency even at the expense of intuition sometimes. 🤷

Reply:
I'm fine with keeping it like this.